src/api.v2/helpers/worker/workSubmit/getExtraDependencies.js

```diff
@@ -6,6 +6,27 @@ const getClusteringSettings = async (message) => {
   return clusteringSettings;
 };
 
+const getSelectedCellSet = async (message, body, cellSets) => {
+  const { selectedCellSet } = body.downsampleSettings;
+
+  const cellSetsKeys = cellSets
+    .find(({ key }) => key === selectedCellSet)
```

Comment:
We have the same file `getExtraDependencies` with the same functions also in the UI. Why is this duplication necessary?